pkg/devcontainer/setup/setup.go

```diff
@@ -234,6 +242,79 @@ func ChownAgentSock(setupInfo *config.Result) error {
 	return nil
 }
 
+func SetupKubeConfig(ctx context.Context, setupInfo *config.Result, tunnelClient tunnel.TunnelClient, log log.Logger) error {
```

Comment:
nit: docblock would be useful, I think a reminder to the caller that it actually writes the file to the user's kube config location would be good